packages/core/src/ceramic.ts

```diff
@@ -90,6 +90,7 @@ export interface CeramicConfig {
   pubsubTopic?: string;
 
   docCacheLimit?: number;
+  concurrentTasksLimit?: number;
```

Comment:
+1 for suggestions.

A bit torn between requests and tasks. `request` has strong connotations to network and I/O, and this is good, but tasks are all over internal code: in the execution queue, in doc-comments, in NamedTaskQueue, so maybe this would serve as an argument for tasks? (Not a strong opinion)

Reply:
My assumption is that most users won't read the internal code, so having a mismatch between the user-visible names and the internal names doesn't bother me as much as making sure that the public-facing names make the most sense.  So just considering the name and its behavior completely independently of the implementation details, "requests" seems slightly better to me. But it's also not a super strong opinion for me either.